src/rules/rule-non-nested-empty-line-before/__tests__/index.js

```diff
@@ -49,6 +49,28 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { except: ["after-single-line-comment"] } ],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "/**\n * comment\n*/\n\na {}",
+  }, {
+    code: "/**\n\t* comment\n*/\na {}",
```

Comment:
See my comment below, I'm pretty sure I've got the tests accurate, its just the rule option I've not got quite right so that's why tests are not doing what's expected 😞